components/StyledComponents/MainContent.js

```diff
@@ -8,14 +8,12 @@ export const MainContent = styled.main`
   align-items: center;
   padding: 0 1rem;
   gap: 1.5rem;
-  margin: var(--header-height-mobile) 0 var(--footer-height) 0;
+  margin: 0 0 var(--footer-height) 0;
 
   z-index: 2;
   width: 100vw;
 
   @media ${device.tablet} {
-    margin-bottom: 0;
-    margin-top: var(--header-height-desktop);
   }
 `;
```

Comment:
i think you can delete this, when it's empty

Reply:
Yes true :) Thanks